File: JavaScript/reverse-words.js
// mine
function reverseWords(str) {
  let result = []
  for (let word of str.split(" ")) {
    result.push(word.split("").reverse().join(""));
  }
  return result.join(" ")
}

// clever
function reverseWords(str) {
  return str.split("").reverse().join("").split(" ").reverse().join(" ");
}

File: Python/reverse_words.py
# mine
def reverse_words(text):
    result = []
    for words in text.split(" "):
        result.append(words[::-1])
    return " ".join(result)

# pythonic
def reverse_words(str):
    return ' '.join(s[::-1] for s in str.split(' '))
